### PR TITLE
Trying to reduce slug size

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: .slugignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/scripts
+/src/TransformCore/PHE/ExporterBundle/Tests
+/src/TransformCore/PHE/HayApi/PersistenceBundle/Tests
+README.md
+LICENSE
+/coverage
```
Comment (eddiejaoude) [inline .slugignore]: what about 

* `*/Resources/doc/`
* `DataFixtures` ?

Comment (brettminnie) [inline .slugignore]: Good Call, also trying to force vendors

On Fri, Nov 13, 2015 at 3:53 PM, Eddie Abou-Jaoude <notifications@github.com

Comment (brettminnie) [inline .slugignore]: Datafixtures is needed here as we use it seed a user

Is a test api

Comment (eddiejaoude) [inline .slugignore]: Force vendors?

Comment (brettminnie) [inline .slugignore]: It was for some reason including the vendors from the travis build, then
the vendors here, added that to slug ignore, we have a working deployment

On Fri, Nov 13, 2015 at 4:08 PM, Eddie Abou-Jaoude <notifications@github.com

Comment (eddiejaoude) [inline .slugignore]: :+1:

**Pull request merged**